__tests__/auth.test.js

```diff
@@ -0,0 +1,5 @@
+describe('Login', () => {
+  describe('Login endpoint', () => {
+    it('should hit the login endpoint', () => {});
```

Comment:
it's not good practice to write empty `it` test blocks - there's no real need to add a test since the controller function doesn't have any logic to test